django/ReleaseAndStage/metadata.yml

```diff
@@ -1,5 +1,10 @@
+suggested: true
```

Comment:
could you please document somewhere the implications of  `suggested` and `tech-preview` ?

Reply:
These changes are part of a new UI see: fabric8-launcher/launcher-planning#122

Reply:
Can you please describe the changes properly in the commit message? Make sure you squash all commits too  and describe  what are you doing without having to refer to another issues repo's

Reply:
Hi chmaouel, As mentioned by erik above, these changes are part of the new getting started UI design. Please refer to the github issue for the change 
https://github.com/fabric8-launcher/launcher-planning/issues/122

Reply:
1. Description for each pipeline stage is added.
2. Suggested is something that is shown in the UI as a suggestion and regarding tech-preview -  if some pipeline they are showing as an experimental feature, they add this tech preview